messages.proto

```diff
@@ -285,6 +285,25 @@ message StepPositionsResponse {
     string error = 2;
 }
 
+/// Request for getting Implementation file list
+message ImplementationFileListRequest {
+}
+
+/// Response for getting Implementation file list
+message ImplementationFileListResponse {
+    /// Get the implementation directory list for folder path
+    repeated string implementationFilePaths = 1;
```

Comment:
Are we getting the directories or all files?

Reply:
we are getting all files

Reply:
Please change the comment then

Reply:
Thanks, missed that